alibi_detect/od/__init__.py

```diff
@@ -23,4 +66,10 @@
     "SpectralResidual",
     "LLR",
     "OutlierProphet"
```

Comment:
This is a bug as a comma is missing after "OutlierProhpet"

Reply:
Good catch. I didn't realise python quietly concatenates strings if the above occurs:

```python
['a', 'b' 'c', 'd'] == ['a', 'bc', 'd']
```